project/make/.ensure-httpserver

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+# Build a Go static web server on top of busybox image
+# and compile it for target daemon
+IMAGE="httpserver"
+
+dir="$DEST/httpserver"
+mkdir -p "$dir"
+(
+	cd "$dir"
+	GOOS=linux GOARCH=amd64 go build -o httpserver github.com/docker/docker/contrib/httpserver
+
+	cat > Dockerfile <<-'EOF'
+		FROM busybox
+		EXPOSE 80/tcp
+		COPY httpserver .
+		CMD ["./httpserver"]
```

Comment:
@tianon fixed. placed Dockerfile in contrib/httpserver/ and copying from there at build time.